src/Support/ClassLoader.php

```diff
@@ -330,7 +427,10 @@ protected static function normalizeClass($class)
         $lowerClass = strtolower($directory) . DIRECTORY_SEPARATOR . $file . '.php';
         $upperClass = $directory . DIRECTORY_SEPARATOR . $file . '.php';
 
-        return [$lowerClass, $upperClass];
+        $lowerClassStudlyFile = strtolower($directory) . DIRECTORY_SEPARATOR . Str::studly($file) . '.php';
+        $upperClassStudlyFile = $directory . DIRECTORY_SEPARATOR . Str::studly($file) . '.php';
+
+        return [$lowerClass, $upperClass, $lowerClassStudlyFile, $upperClassStudlyFile];
```

Comment:
What was this change for?

Reply:
@LukeTowers It was to catch some extra permutations of the filename - essentially allow for PSR-4 filenames. It was mainly for the Backend controller class, which searches a lower-case filename when determining the controller from the URL.